ccli.gemspec

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+lib = File.expand_path('lib', __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+
+Gem::Specification.new do |s|
+  s.name          = 'ccli'
+  s.version       = '0.1.0'
+  s.summary       = 'CLI tool to manage Openshift Secrets via Cryptopus'
```

Comment:
Command line client for the opensource password manager Cryptopus ?

probably it's best to just make this summary pretty generic ... hopefully it will be extended in the future